frontend/app/src/components/defi/Borrowing.vue

```diff
@@ -76,7 +76,7 @@
                   aspect-ratio="1"
                   contain
                   position="left"
-                  max-width="48px"
+                  max-width="26px"
                   max-height="24px"
```

Comment:
Any particular reason these are 24x26 instead of 24x24 (or I guess 26x26)?

Reply:
Yes, the image was not displaying as a proper circle for some reason. Part of the edge was getting cut.

Reply:
Ok!